students/jeewon-shin/westagram/user/views.py

```diff
@@ -0,0 +1,36 @@
+import json
+import bcrypt
+
+from django.http  import JsonResponse
+
+from django.views import View
+from user.models  import User
+
+class SignupView(View):
+    def post(self, request):
+        PASSWORD_LENGTH = 8
+        try:
+            data = json.loads(request.body)
+            hashed_password = bcrypt.hashpw(data['password'].encode('utf-8'), bcrypt.gensalt())
+
+            if '@' not in data['email']:
+                return JsonResponse({'MESSGAGE': 'INVALID EMAIL'}, status=400)
+            elif '.' not in data['email']:
+                return JsonResponse({'MESSGAGE': 'INVALID EMAIL'}, status=400)
```

Comment:
```suggestion
            if '@' not in data['email'] or '.' not in data['email']::
                return JsonResponse({'MESSGAGE': 'INVALID EMAIL'}, status=400)
```
or 로 조건을 묶을 수 있지 않을까요?